TimeIsMoney/TimeIsMoneyMasterViewController.h

```diff
@@ -7,23 +7,31 @@
 //
 
 #import <UIKit/UIKit.h>
+#import "AppDelegate.h"
 
-#import "TimeIsMoneySettingsModel.h"
+typedef enum timerState
+{
+    STOPPED,
+    PAUSED,
+    RUNNING_TASK,
+    RUNNING_BREAK,
+} TimerState;
```

Comment:
There might be an easier way to declare these - and timerState and TimerState should be the same?